src/containers/AccountModal/AccountModal.tsx

```diff
@@ -73,11 +73,10 @@ const AccountModal = () => {
         </div>
       );
     }
+
     switch (view) {
-      case 'simultaneous-logins':
-        return <SimultaneousLoginsNotification />;
       case 'login':
-        return <Login />;
+        return <Login message={message as LoginFormMessage} />;
```

Comment:
I agree with the sentiment, but would recommend a slightly different solution. Since we probably want to put the translation tags all into a switch statement anyhow so the i18n parser picks them up properly, why don't we let the message get passed as a string into the Login component and then have a switch statement in there like this:
```
const getMessage = (messageId: string) => {
    if (messageId === 'simultaneous_warning') {
      return t('simultaneous_logins');
    }

    return t('unexpected_login_error');
  }
```

Reply:
Yes, I also like that because you won't need to use the `LoginFormMessage` type in several places.

The only misaligned could be that the string literal is not set correctly in the NotificiationsController, but I think you already mentioned (in a different thread) that using a generic constant could solve this problem:

```ts
export const simultaneousLoginsFlag = 'simultaneous_logins';
```